lib/src/collect.dart

```diff
@@ -97,6 +97,8 @@ Future<Map<String, dynamic>> _getAllCoverage(
   scopedOutput ??= <String>{};
   final vm = await service.getVM();
   final allCoverage = <Map<String, dynamic>>[];
+  final ver = await service.getVersion();
```

Comment:
[nit] 
```suggestion
  final version = await service.getVersion();
```
https://dart.dev/guides/language/effective-dart/design#avoid-abbreviations

Reply:
Done